content/04.study.md

```diff
@@ -47,6 +47,25 @@ Deep learning applied to gene expression data is still in its infancy, but the f
 Many previously untestable hypotheses can now be interrogated as deep learning enables analysis of increasing amounts of data generated by new technologies.
 For example, the effects of cellular heterogeneity on basic biology and disease etiology can now be explored by single-cell RNA-seq and high-throughput fluorescence-based imaging, techniques we discuss below that will benefit immensely from deep learning approaches.
 
+### DNA Methylation
+
+#### Introduction 
+
+DNA Methylation (DNAm), is the process of adding a methyl group to a cytosine in the context of a CpG dinucleotide. 
+This DNA-level epigenetic modification regulates gene transcription, is critical in development, and alterations to DNA methylation are well-established as contributing to pathophysiology of many diseases including cancers. 
+Studies of DNA methylation have also shed light on the processes involving initial differentiation of stem cells, aging and pathogenesis in response to environmental exposures. 
```

Comment:
References would help here, again - over-reference the bio side to help ML people get to relevant literature quickly.